src/plugins/getCurrentTimeStamp.js

```diff
@@ -0,0 +1,4 @@
+export const getCurrentTimeStamp = () => {
+  const date = new Date();
+  return Math.floor(date.getTime() / 1000);
+};
```

Comment:
I assume you should do this (`export default`),
so you can import it as `import getCurrentTimeStamp from '../plugins/getCurrentTimeStamp';` on https://github.com/taishikato/shouter/blob/feature_postFunction/src/components/PostForm.jsx#L5
```javascript
const getCurrentTimeStamp = () => {
  const date = new Date();
  return Math.floor(date.getTime() / 1000);
};
export default getCurrentTimeStamp
```

If you just do like `export const getCurrentTimeStamp = () => {`, you need to import it as `import { getCurrentTimeStamp } from '../plugins/getCurrentTimeStamp';` as long as I know.